static/js/components/EducationTab.js

```diff
@@ -88,79 +90,88 @@ class EducationTab extends ProfileTab {
     setEducationDialogIndex(newIndex);
     setEducationDegreeLevel(level);
     setEducationDialogVisibility(true);
-  }
+  };
 
   deleteEducationEntry = index => {
     const { updateProfile, profile } = this.props;
     let clone = _.cloneDeep(profile);
     clone['education'].splice(index, 1);
     updateProfile(clone);
-  }
+  };
 
-  printAddDegree(level){
-    const { ui: { educationDegreeInclusions } } = this.props;
-    if (educationDegreeInclusions[level.value]){
-      let openForm = () => this.openNewEducationForm(level.value, null);
-      return <Grid key={`add-${level.value}`}>
-        <Cell col={11}></Cell>
-        <Cell col={1}>
-          <FABButton mini onClick={openForm} raised ripple>
-            <Icon name="add" />
-          </FABButton>
-        </Cell>
-      </Grid>;
+  renderEducationLevel(level){
+    const { ui: { educationDegreeInclusions }, profile: { education } } = this.props;
+    if (educationDegreeInclusions[level.value]) {
+      let rows = [];
+      if (education !== undefined) {
+        rows = Object.entries(education).
+          filter(([index, education]) => education.degree_name === level.value).  // eslint-disable-line no-unused-vars
```

Comment:
perhaps I'm misunderstanding something, but why is `Object.entries` necessary here? Isn't `profile.education` an array? Why could we do something like:

```js
rows = education.filter( education => education.degree_name === level.value ).
map( ... ) 
```